worker/src/request_init.rs

```diff
@@ -281,49 +297,134 @@ pub struct MinifyConfig {
-    }
+#[derive(Clone, serde::Serialize)]
+#[serde(untagged)]
+pub enum ResizeBorder {
+    Uniform {
+        color: String,
+        width: usize,
+    },
+    Varying {
+        color: String,
+        top: usize,
+        right: usize,
+        bottom: usize,
+        left: usize,
+    },
 }
 
-impl From<PolishConfig> for &str {
-    fn from(conf: PolishConfig) -> Self {
-        match conf {
-            PolishConfig::Off => "off",
-            PolishConfig::Lossy => "lossy",
-            PolishConfig::Lossless => "lossless",
-        }
-    }
+/// Configuration options for Cloudflare's image resizing feature:
+/// <https://developers.cloudflare.com/images/image-resizing/>
+#[derive(Clone, Default, serde::Serialize)]
+pub struct ResizeConfig {
+    pub anim: Option<bool>,
+    pub background: Option<String>,
+    pub blur: Option<u8>,
+    pub border: Option<ResizeBorder>,
+    pub brightness: Option<f64>,
+    pub compression: Option<ResizeCompression>,
+    pub contrast: Option<f64>,
+    pub dpr: Option<f64>,
+    pub fit: Option<ResizeFit>,
+    pub format: Option<ResizeFormat>,
+    pub gamma: Option<f64>,
+    pub gravity: Option<ResizeGravity>,
+    pub height: Option<usize>,
+    pub metadata: Option<ResizeMetadata>,
+    pub onerror: Option<ResizeOnerror>,
+    pub quality: Option<usize>,
+    pub rotate: Option<usize>,
+    pub sharpen: Option<usize>,
+    pub trim: Option<ResizeTrim>,
+    pub width: Option<usize>,
+}
+
+#[derive(Clone, Copy, Serialize)]
+#[serde(rename_all = "kebab-case")]
+pub enum ResizeCompression {
+    Fast,
+}
+
+#[derive(Clone, Copy, Serialize)]
+#[serde(rename_all = "kebab-case")]
+pub enum ResizeFit {
+    ScaleDown,
+    Contain,
+    Cover,
+    Crop,
+    Pad,
+}
+
+#[derive(Clone, Copy, Serialize)]
+#[serde(rename_all = "kebab-case")]
+pub enum ResizeFormat {
+    Auto,
+    Avif,
+    Webp,
+    Json,
+}
+
+#[derive(Clone, Copy, Serialize)]
+#[serde(rename_all = "kebab-case")]
+pub enum ResizeGravitySide {
+    Auto,
+    Left,
+    Right,
+    Top,
+    Bottom,
+}
+
+#[derive(Clone, Copy, Serialize)]
+#[serde(rename_all = "kebab-case")]
+#[serde(untagged)]
+pub enum ResizeGravity {
+    Side(ResizeGravitySide),
+    Coords { x: f64, y: f64 },
+}
+
+#[derive(Clone, Copy, Serialize)]
+#[serde(rename_all = "kebab-case")]
+pub enum ResizeMetadata {
+    Keep,
+    Copyright,
+    None,
 }
 
-#[wasm_bindgen]
-#[derive(Default, Clone, Copy)]
+#[derive(Clone, Copy, Serialize)]
+#[serde(rename_all = "kebab-case")]
+pub enum ResizeOnerror {
+    Redirect,
+}
+
+#[derive(Clone, Copy, Serialize)]
+#[serde(rename_all = "kebab-case")]
+pub struct ResizeTrim {
+    pub top: usize,
+    pub bottom: usize,
+    pub left: usize,
+    pub right: usize,
+    pub width: usize,
+    pub height: usize,
+}
+
+#[derive(Clone, Copy, Default, Serialize)]
+#[serde(rename_all = "kebab-case")]
 pub enum RequestRedirect {
     Error,
     #[default]
     Follow,
     Manual,
 }
 
-impl From<RequestRedirect> for &str {
```

Comment:
I know these `Into<&str>` impls aren't necessary anymore, but it'd be nice to not introduce any breaking changes however unlikely they might actually be.